phpunit/class-wp-theme-json-resolver-test.php

```diff
@@ -250,7 +250,7 @@ function test_merges_child_theme_json_into_parent_theme_json() {
 		);
 
 		$this->assertSame(
-			WP_Theme_JSON_Resolver_Gutenberg::get_theme_data()->get_custom_templates(),
+			WP_Theme_JSON_Resolver::get_theme_data()->get_custom_templates(),
```

Comment:
As above, this line shouldn't be reverted, as the test should be for the Gutenberg version of the class.